crypto/build.info

```diff
@@ -74,7 +74,7 @@ DEFINE[../providers/libfips.a]=$CPUIDDEF
 # already gets everything that the static libcrypto.a has, and doesn't need it
 # added again.
 IF[{- !$disabled{module} && !$disabled{shared} -}]
-  SOURCE[../providers/liblegacy.a]=$CPUID_COMMON
+  SOURCE[../providers/liblegacy.a]=$CPUID_COMMON param_trie.c
```

Comment:
Try this instead:

```
  SOURCE[../providers/libcommon.a]=param_trie.c
```

Reply:
(this has me wonder if `$CPUID_COMMON` shouldn't end up in libcommon.a as well...  things have moved around so many times that I lost track...  BUT, that's for another PR)

Reply:
I see you made the change, and it seems to have actually helped